test/00/groupResize.sh

```diff
@@ -46,8 +46,9 @@ proc afterMinskyStarted {} {uplevel #0 {
  canvas.mouseDown [expr \$x+0.5*\$w] [expr \$y+0.5*\$h]
  canvas.mouseUp [expr \$x+\$w]  [expr \$y+\$h]
 
- assert "abs([expr 1.5*\$w]-[minsky.canvas.item.width])<5"
- assert "abs([expr 1.5*\$h]-[minsky.canvas.item.height])<5"
+ # Resize handles larger on groups, see ticket 1023, so needed to adjust the factors below
```

Comment:
Yes - this. I'm still unhappy with the GroupItem placement of resize handles.

Reply:
I realised after I requested the merge that I still hadn't fixed up the test, sorry. Once you have implemented the alternative to the Accessor, I will fix it if still required. I think the problem is caused by a combination of the larger resize handles and the left and right margins of the IO variables, which I still have to figure out how to call in the test TCL code, but haven't yet managed to.